packages/@ember/-internals/routing/lib/system/route.ts

```diff
@@ -2082,11 +2082,21 @@ export function getFullQueryParams(router: EmberRouter, state: TransitionState<R
     return state['fullQueryParams'];
   }
 
-  state['fullQueryParams'] = {};
-  Object.assign(state['fullQueryParams'], state.queryParams);
+  let fullQueryParamsState = {};
+  let haveAllRouteInfosResolved = state.routeInfos.every((routeInfo) => routeInfo.route);
```

Comment:
The master branch no longer supports IE11 (which is why `.every` works in CI here), but if we are going to backport to 3.24 (current LTS) then we'll need to migrate to be IE11 compatible.

Reply:
Looking at [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/every#browser_compatibility), I think `Array.prototype.every` was initially part of ES5 and rolled out in IE9. For some reason, I also thought that `.every` was much newer, but it seems like this _should_ work in IE11.

Reply:
Apparently, I'm bonkers. `.every` **was** in IE9...